frontend/packages/topology/src/components/quick-search/QuickSearchModal.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+import QuickSearchModalBody from './QuickSearchModalBody';
+import { CatalogItem } from '@console/plugin-sdk';
+
+interface QuickSearchModalProps {
+  isOpen: boolean;
+  namespace: string;
+  closeModal: () => void;
+  allCatalogItemsLoaded: boolean;
+  searchCatalog: (query: string) => CatalogItem[];
+}
+
+const QuickSearchModal: React.FC<QuickSearchModalProps> = ({
+  isOpen,
+  namespace,
+  closeModal,
+  searchCatalog,
+  allCatalogItemsLoaded,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Modal
+      variant={ModalVariant.medium}
+      aria-label={t('topology~Quick Search')}
+      isOpen={isOpen}
+      showClose={false}
+      position="top"
+      positionOffset="30%"
```

Comment:
🤔  Is this new... because that's probably going to cause wonky stuff on Tablet-sized devices when the nav section collapses.

Reply:
Are you talking about the `positionOffset`?

Reply:
I added this because according to UXD the modal is neither at the top nor in the middle that is why had to add an offset.

Reply:
![Peek 2020-11-30 23-02](https://user-images.githubusercontent.com/20724543/100643192-be25f380-335f-11eb-987d-08324a1a5864.gif)
 This is how it looks when the nav collapses.

Reply:
Oh, it's a top offset, not a left offset. Apologizes!